src/components/calculator.js

```diff
@@ -0,0 +1,38 @@
+function Calculator() {
+  function ButtonDiv() {
+    return (
+      <div className="btnDiv">
+        <button type="button" className="operator">AC</button>
+        <button type="button" className="operator">+/-</button>
+        <button type="button" className="operator">%</button>
+        <button type="button" className="operator">/</button>
+        <button type="button" className="number">7</button>
+        <button type="button" className="number">8</button>
+        <button type="button" className="number">9</button>
+        <button type="button" className="operator">x</button>
+        <button type="button" className="number">4</button>
+        <button type="button" className="number">5</button>
+        <button type="button" className="number">6</button>
+        <button type="button" className="operator">-</button>
+        <button type="button" className="number">1</button>
+        <button type="button" className="number">2</button>
+        <button type="button" className="number">3</button>
+        <button type="button" className="operator">+</button>
+        <button type="button" className="number zero">0</button>
+        <button type="button" className="number">.</button>
+        <button type="button" className="operator equal">=</button>
+      </div>
+    );
+  }
```

Comment:
- [ ] [OPTIONAL] Consider placing the `ButtonDiv` component in a separate file to promote reusability and adhere to React's best practices. By moving the `ButtonDiv` component to its own file, you can enhance code organization and maintainability. It allows the component to be easily imported and reused in multiple parts of your application, reducing duplication and improving modularity.
- This practice aligns with React's component-based architecture, where each component should have a clear responsibility and be independent of the surrounding code. Separating the `ButtonDiv` component into its own file enhances code readability and makes it more manageable in the long run.